src/libmongoc/src/mongoc/mongoc-config.h.in

```diff
@@ -353,13 +365,18 @@
 /*
  * Set if tracing is enabled. Logs things like network communication and
  * entry/exit of certain functions.
- *
  */
-#define MONGOC_TRACE @MONGOC_TRACE@
+#cmakedefine01 MONGOC_TRACE
```

Comment:
This unexpectedly broke PHPC's build system (autotools). I've reported [CDRIVER-4247](https://jira.mongodb.org/browse/CDRIVER-4247) to request a more portable approach.

Reply:
Resolved in https://github.com/mongodb/mongo-c-driver/pull/917.